code/modules/projectiles/gun.dm

```diff
@@ -212,6 +214,7 @@
 				shoot_with_empty_chamber(user)
 				break
 			process_chamber()
+			process_laser(target, user, params)
```

Comment:
It felt weird re-using pointer_icon_state from laserpointer.dm but I will make that correction.